### ISSUE TITLE
18199194813@163.com

### ISSUE THREAD

**Issue by rabbit218218** [LMD0311/PointMamba#30]
I am currently facing a few issues:
(1) What is the purpose of pretraining?
(2) In the model section of PointMamba/cfgs/finetune_modelnet.yaml, the parameter NAME:PointMamba, which indicates that the PointMamba module from point_mamba.py is being used. However, it seems that no other sections, such as maskmamba, are utilized. Does this mean that when classifying the ModelNet40 dataset, there is no process of masking and reconstruction?

**Comment by LMD0311** [comment]
Thank you for your interest in our work. Mask modeling-based pre-training is commonly utilized to enhance performance, with the pre-trained weights serving as a starting point for fine-tuning downstream tasks. We use `MaskMamba` for pre-training, maintaining the same structure for fine-tuning.

**Comment by rabbit218218** [comment]
![modelnet40](https://github.com/user-attachments/assets/c53af19c-ac3f-4ad6-b4a4-018e152152a4)
![pretrain](https://github.com/user-attachments/assets/909d8050-7060-451a-ac66-9dd29fbea078)
I noticed that the models used in the two .yaml files are different, so the model loaded into the network at runtime will be different, right? However, I've noticed that neither of these models mentions the use of the maskmamba module, and I now want to use the maskmamba module, but I don't know how to do it. Additionally, I also need your help to answer a question about Loss. I want to know what the two values in Losses=['data1','data2'] mean in the log after the network has been trained (using ModelNet40 as an example). If you have time, I would appreciate it if you could reply. Thank you very much.

**Comment by rabbit218218** [comment]
Could you please reply to this message? I'm really eager to learn about the related work.

**Comment by LMD0311** [comment]
Apologies for the delayed response. Because there are some differences between pre-training and fine-tuning forwards, they are written in two Classes. But the naming of the key is the same for both. The structure of the code is almost the same as Point-MAE.
